swan-lake/integration-tools/health-tool.md

```diff
@@ -420,5 +421,226 @@ Follow the steps below to use the generated API templates by running the cloned
+
+### Template generation usage
+
+```
+$ bal health fhir
+            [--org-name] <template-organization-name>
+            [--package-name] <fully-qualified-name-of-package>
+            [--package-version] <version-of-the-package>
+            [-o | --output] <output-location>
+            [-i | --input] <cds-hook-definitions-file-path>
+```
+
+### CDS template generation command options
+
+| Command option      | Description                                                                                                                                                                                                                                                                                                                                                                                                                           | Mandatory/Optional |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------ |
+| `--org-name`        | Organization name of the Ballerina template to be generated. For more information, see <a href="https://ballerina.io/learn/package-references/#the-org-field" target="_blank"> the <code>org</code> field</a>.                                                                                                                                                                                                         | Optional           |
+| `--package-name`    | Name of the Ballerina package to be generated. The package name can be explicitly set using this argument. Unless specified, the default name `health.fhir.templates.crd` will be taken to construct the name of the package.                                                                                                                                                                                                         | Optional           |
+| `--package-version` | The version of the Ballerina template to be generated.                                                                                                                                                                                                                                                                                                                                                                                | Optional           |
+| `-i, --input`       | Only the TOML file type is acceptable. Users need to define the CDS hook definitions in the input file. The given CDS hooks input will be used to generate the Ballerina service. | Mandatory          |
```

Comment:
shall we keep Optional fields together?

Reply:
Updated: https://github.com/ballerina-platform/ballerina-dev-website/pull/9636/commits/f64bf61fdea84f1436f789c8637eacf16e432475